app/src/main/java/jp/developer/retia/frozenword/ui/habbits/HabbitsActivity.kt

```diff
@@ -57,5 +64,12 @@ class HabbitsActivity : ComponentActivity() {
 fun HabbitsScreen(habbitsViewModel: HabbitsViewModel) {
     val state by habbitsViewModel.uiState.collectAsState()
 
-    HabbitsScreen(state)
+    Scaffold(floatingActionButton =
+    {
+        FloatingActionButton(onClick = { habbitsViewModel.onClickActionButton() }) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/verno3632/FrozenWord/blob/784faffd28f4924cc6692482eff358e508b727bb/app/src/main/java/jp/developer/retia/frozenword/ui/habbits/HabbitsActivity.kt#L69"></span>Unexpected indentation (8) (should be 12)</td>
    </tr>
  </tbody>
</table>